### DISCUSSION TITLE
Preserve the size and resolution of a pdf file

### DISCUSSION THREAD

**Discussion by manuel-collab** [dlemstra/Magick.NET#1655] [Help]
How can I preserve the characteristics of a document by performing the following command?

convert color.pdf color_converted.pdf

When I open color_converted.pdf I notice that the resolution of the images it contains has been modified. How can I convert the file without it being altered? Thanks

**Comment by dlemstra** [comment]
Why are you asking that question here? This looks like an ImageMagick question?

**Comment by manuel-collab** [comment]
I was referring in Magick.NET. Thank you


Manuel Quero Delgado

**Comment by dlemstra** [comment]
I don't think my library is the best library to use when converting from pdf to pdf. Is it possible to share your file so I can give more information about why this is happening?
